nemo_skills/prompt/config/generic/codegen_system.yaml

```diff
@@ -0,0 +1,13 @@
+# Default prompt that does not contain any extra instructions. Can be used for instruction following evals (e.g. ifeval)
```

Comment:
Comment is not up-to-date. Also should we replace the original prompt with this one instead of keeping both?

Reply:
Hmm actually its not working yet, so i don't think its worth upstreaming just yet.